addon/components/hyper-table-v2/index.hbs

```diff
@@ -23,7 +23,7 @@
           {{/if}}
         </div>
         <div class="right-side">
-          {{#if this.features.global_filters_reset}}
+          {{#if (and this.features.global_filters_reset this.displayResetButton)}}
```

Comment:
nit: move the `this.features.global_filters_reset` check into `displayResetButton` directly